docs/active-directory/core-concepts/01-active-directory-details-and-definitions.md

```diff
@@ -0,0 +1,111 @@
+# Active Directory Details and Definitions 
+
+### Active Directory Domain Services
+
+Active Directory Domain Services (AD DS, or more simply just AD for Active Directory) is a storage and management
+mechanism for everything related to a collection of resources within a computer network. This includes users, files,
+network configurations, and devices like physical computers and printers. Administrators can modify the Active Directory
+and remotely manage all the components currently tracked at once. In its simplest terms, **AD is used to logically
+represent an organizations resources and users and enforce policies and behaviors across them.**
+
+AD Includes the following features
```

Comment:
```suggestion
AD Includes the following features:
```

Reply:
In general, I think it's easier to read the docs if we just replace AD -> Active Directory.